### DISCUSSION TITLE
padding on Contributor h3 headline

### DISCUSSION THREAD

**Discussion by ciruz** [community/community#5994] [Other Feature Feedback, Questions, & Ideas] [answered]
I just checked out the new Release Note design, it's cool but I don't like the padding between the ``Contributors`` headline and the user avatars. The padding looks not right to me.

Compare the padding from the headlines in the release notes markdown text (``Core Changes``, ``Documentation C...``) to the padding between ``Contributors`` and the user avatars, and compare it the padding between user avatars and the reaction buttons (emoji, like).

State right now:
![image](https://user-images.githubusercontent.com/987947/136019933-c54d5b55-ebc9-4cbc-b56f-7f14971c3661.png)

Improved idea:
![image](https://user-images.githubusercontent.com/987947/136020401-aaa97b2a-4b82-4db3-9759-b6674228a40d.png)

**Comment by MylesBorins** [comment, marked as answer]
Thanks for the feedback @ciruz. We will be doing another pass on design and I'll let the team know about this

**Reply by MylesBorins** [reply]
as an update we shipped a fix this week

**Reply by ciruz** [reply]
cool thanks
